inference-engine/src/cldnn_engine/cldnn_common_utils.h

```diff
@@ -9,20 +9,13 @@
 #include <cpp_interfaces/exception2status.hpp>
 #include <api/layout.hpp>
 
+#include "ngraph/ngraph.hpp"
```

Comment:
BTW, compilation process will be faster if you will use dedicatde header file in each `ops/*.cpp` file instead of this global one.

Reply:
Good idea. Removed `#include <ngraph/ngraph.hpp>` from headers. Ops now include only dedicated headers, cldnn_program.h uses forward declaration of ngraph::Node

Reply:
thanks!